presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/PushPartialAggregationThroughExchange.java

```diff
@@ -100,6 +100,11 @@ public Result apply(AggregationNode aggregationNode, Captures captures, Context
             return Result.empty();
         }
 
+        // currently, pushing partial aggregation though merging exchange is not supported
```

Comment:
typo: though -> through

Reply:
Excluding streaming aggregations from this rule caused a regression. How are you thinking about why is it safe to exclude merging exchanges?

Reply:
Streaming aggregations are excluded in pattern (as for now):
```
    private static final Pattern<AggregationNode> PATTERN = aggregation()
            .matching(node -> !node.isStreamable())
            .with(source().matching(exchange().capturedAs(EXCHANGE_NODE)));
```

Otherwise I think we can safety push them through merging exchange since they don't change order of pre-grouped symbols